jest.config.js

```diff
@@ -131,7 +133,7 @@ module.exports = {
   // snapshotSerializers: [],
 
   // The test environment that will be used for testing
-  testEnvironment: "node",
+  testEnvironment: "jsdom",
```

Comment:
not sure if this is a typo meant to be "json", or if actually intended to be "jsdom"

Reply:
jsdom means js with dom simulation :)